bin/run

```diff
@@ -36,9 +37,15 @@ async function main() {
   } catch(err) {
     await require('@oclif/errors/handle')(err);
 
-    console.log(err.stack);
-    console.log(err.code);
     console.log(err.message);
+
```

Comment:
I suggest this is reverted. The error message printed here is important information for the user. We should not require a debug flag to be set for them to see it.